source/_integrations/discord.markdown

```diff
@@ -21,6 +21,7 @@ To use Discord notifications, add the following to your `configuration.yaml` fil
 # Example configuration.yaml entry
 notify:
   - platform: discord
+    name: Some_Name
```

Comment:
It should work without, if not, please raise an issue with the core repository for the Discord integration.

We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample (and `name` is optional). This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.